cleanlab/token_classification/summary.py

```diff
@@ -143,6 +144,7 @@ def display_issues(
             print(color_sentence(sentence, word))
         else:
             shown += 1
+            # issue is tuple[int, int] but gets treated as int.
             sentence = get_sentence(tokens[issue])
             print("Sentence %d: %s" % (issue, sentence))
```

Comment:
Dont know what to do with those few lines.

Reply:
Yeah, we expect `issue` to be inferred as an `int` if we reach this block of code.
The signature of `display_issues` isn't correct here. It should accept either a list of integers or a list of pairs of integers.

Ultimately, this function can be refactored but I think it's outside the scope of this PR.

The sensible step is to just leave it as is and get the current changes working.
We can always make a new PR for ongoing work.